cmd/skupper/README.md

```diff
@@ -0,0 +1,41 @@
+# Skupper CLI
+
+The [Skupper](https://skupper.io/) command line enables you to:
+
+* Create a Virtual Application Network (VAN) site
+* Connect to other VAN sites
+* Connect services
+
+## Getting `skupper`
+
+To install `skupper`, download the [latest release](https://github.com/skupperproject/skupper/releases).
```

Comment:
@ajssmith not sure if this is how users will get cli?

Reply:
You could point them here: https://skupper.io/start/

As I believe most of the examples have been updated to reference this narrative.